GameBoard/Views/Shared/_Layout.cshtml

```diff
@@ -21,20 +21,19 @@
     <environment include="Development">
         <script src="~/lib/jquery/dist/jquery.js"></script>
         <script src="~/lib/popper.js/dist/umd/popper.js"></script>
-        <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css"/>
-        <link rel="stylesheet" href="~/lib/font-awesome/css/font-awesome.css"/>
-        <link rel="stylesheet" href="~/lib/malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.css"/>
-        <link rel="stylesheet" href="~/lib/tempusdominus-bootstrap-4/build/css/tempusdominus-bootstrap-4.css"/>
+        <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.css" />
+        <link rel="stylesheet" href="~/lib/font-awesome/css/font-awesome.css" />
+        <link rel="stylesheet" href="~/lib/malihu-custom-scrollbar-plugin/jquery.mCustomScrollbar.css" />
+        <link rel="stylesheet" href="~/lib/tempusdominus-bootstrap-4/build/css/tempusdominus-bootstrap-4.css" />
 
-        <link rel="stylesheet" href="~/css/friendsSidebar.css"/>
-        <link rel="stylesheet" href="~/css/gameEvent.css"/>
-        <link rel="stylesheet" href="~/css/gameEventInviteForm.css"/>
-        <link rel="stylesheet" href="~/css/gameEventForm.css"/>
-        <link rel="stylesheet" href="~/css/gameEventsList.css"/>
-        <link rel="stylesheet" href="~/css/usersSearchBox.css"/>
-        <link rel="stylesheet" href="~/css/popovers.css"/>
-        <link rel="stylesheet" href="~/css/userCard.css"/>
-        <link rel="stylesheet" href="~/css/main.css"/>
+        <link rel="stylesheet" href="~/css/friendsSidebar.css" />
+        <link rel="stylesheet" href="~/css/gameEvent.css" />
+        <link rel="stylesheet" href="~/css/gameEventForm.css" />
+        <link rel="stylesheet" href="~/css/gameEventsList.css" />
+        <link rel="stylesheet" href="~/css/usersSearchBox.css" />
+        <link rel="stylesheet" href="~/css/popovers.css" />
+        <link rel="stylesheet" href="~/css/userCard.css" />
+        <link rel="stylesheet" href="~/css/main.css" />
     </environment>
     <environment exclude="Development">
         <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"
```

Comment:
Why all this changes? It's against our code style policies to put spaces before `/>`.

Reply:
Well, I don't know what happened. Everything seems to have spaces before /> on my computer and on github as well.

Reply:
It does -- and it shouldn't. My R# removes these spaces on Reformat Code.

Reply:
Removed these spaces and the ones in the RemoveFromGameEventForm.